apps/site/src/views/Home/components/ApplyButton/ApplyButton.tsx

```diff
@@ -6,8 +6,9 @@ export default function ApplyButton() {
 	return (
 		<Button
 			className={styles.applyButton}
-			href="https://hack.ics.uci.edu/" // TODO: Change to application site link
+			href="https://forms.gle/nxRU44y9Gd5mrmXUA"
```

Comment:
Now that we have the redirect, we can link to `/apply` to remove the duplicate reference.

Reply:
oh i see, I'll make this change rn.